<launch>
	<arg name="marker_size" default="4.7" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/kinect2/depth_lowres/points" />
	<arg name="cam_info_topic" default="/kinect2/depth_lowres/camera_info" />
	<arg name="output_frame" default="/kinect2_link" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
</launch>
